csrc/ops/composite.cpp

```diff
@@ -106,6 +107,30 @@ TensorView* _matmul_tt(TensorView* a, TensorView* b) {
   return tv2;
 }
 
+TensorView* matmul(TensorView* a, TensorView* b) {
+  NVF_CHECK(
+      a->nDims() == b->nDims(),
+      "The number of dimension of A and B do not match");
+  // TODO: We'll need to suppor nDims == 3 for bmm.
+  NVF_CHECK(
+      a->nDims() == 2,
+      "Only 2-D Tensors are supported, in the future we'll support 3-D as well!");
+  NVF_CHECK(a->getDataType().value() == b->getDataType().value());
+
+  // This flag being set means the matmul is executed via ATen.
+  EnableOptionsGuard::getCurOptions().set(EnableOption::MatmulExprEval);
```

Comment:
This feels like a hack. Are we expecting people to use nvfuser matmul via python API to `always` execute it via aten?

Reply:
This should be resolved after this PR which made execution via ATen default.